lib/modules/cardiovascular_disease.rb

```diff
@@ -366,16 +407,72 @@ def self.perform_encounter(entity, time)
             patient.condition(diagnosis, time, :condition, :condition)
           end
         end
+
+        if entity[:careplan] && entity[:careplan][:cardiovascular_disease]
+          if !entity.record_synthea.careplan_active?(:cardiovascular_disease)
+            entity.record_synthea.careplan_start(:cardiovascular_disease, entity[:careplan][:cardiovascular_disease]['activities'], time, entity[:careplan][:cardiovascular_disease]['reasons'])
+          else 
+            entity.record_synthea.update_careplan_reasons(:cardiovascular_disease, entity[:careplan][:cardiovascular_disease]['reasons'],time)
+          end
+        elsif entity.record_synthea.careplan_active?(:cardiovascular_disease)
+          entity.record_synthea.careplan_stop(:cardiovascular_disease,time)
+        end
+
+        if entity[:medications]
+          [:clopidogrel, :simvastatin, :amlodipine, :nitroglycerin, :atorvasatatin, :captopril, :warfarin, 
+            :verapamil, :digoxin, :epinephrine, :amiodarone, :atropine].each do |med|
+            if entity[:medications][med]
```

Comment:
Is the above just a repetition (and aggregation) of all the meds that could have been prescribed in the rules of the module?  If so, consider if there is a better way to do this that is less repetitious and less error prone.  As it is now, we need to make sure we keep these lists in sync.  Potential approaches might involve using constants, having the module keep track of its own prescribed medications, dynamically looking at medications and reasons to get the list, etc.

All that said, if the solution is overly complex, then... just keep this all as is. ;-)